.github/workflows/build-unitycloud.yml

```diff
@@ -160,9 +176,10 @@ jobs:
           POLL_TIME: 60 # In seconds (int)
           TARGET: t_${{ matrix.target }}
           BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
-          USE_CACHE: ${{ github.event.inputs.cache || inputs.cache }}
+          USE_CACHE: ${{ github.event.inputs.cache || inputs.cache || true}}
```

Comment:
This will always default to true. If I understand correctly, there is no way to build not using cache; and I AFAIK we builld the weekly release with no cache.

Also, do you need both ways of input (`github.event.inputs.cache` and `inputs.cache`)? Not completely sure, but I think using `inputs.cache` is enough.

Reply:
Good catch, actually left that from debugging :D